### PR TITLE
Feature/bbb 405 password grant flow

### PR DESCRIPTION
**Pull request by stephen-bealine**

### CODE REVIEW

File: src/main/java/uk/gov/dft/bluebadge/webapp/la/config/SpringSecurityConfig.java [hunk 1]
```diff
@@ -1,10 +1,70 @@
...
 @Configuration
-@EnableOAuth2Sso
+@EnableOAuth2Client
 @Order(52)
-public class SpringSecurityConfig {}
+public class SpringSecurityConfig extends WebSecurityConfigurerAdapter {
+  @Autowired OAuth2ClientContext oauth2ClientContext;
+
+  @Bean
+  OAuth2RestTemplate restTemplate(ResourceOwnerPasswordResourceDetails resourceDetails) {
```
Comment (mgilgar) [inline src/main/java/uk/gov/dft/bluebadge/webapp/la/config/SpringSecurityConfig.java]: Is this the rest template we will be using from the other clients?

Comment (stephen-bealine) [reply src/main/java/uk/gov/dft/bluebadge/webapp/la/config/SpringSecurityConfig.java]: Yes. Once we have secured the api apps

File: src/main/resources/static/css/main-ie8.css [hunk 2]
```diff
@@ -111,7 +111,7 @@ input, select {
   box-sizing: border-box;
 }
 
-.govuk-link, .dft-search__clear-link, .govuk-prose-scope a {
+.govuk-link, .dft-btn-link, .dft-search__clear-link, .govuk-prose-scope a {
   font-family: "nta", Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
```
Comment (samquayle) [inline src/main/resources/static/css/main-ie8.css]: I'd suggest targeting all elements with these properties

File: src/main/resources/static/css/main-ie8.css [hunk 3]
```diff
@@ -4705,6 +4705,27 @@ body {
...
 
+.dft-btn-link {
+  font-family: "nta", Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-family: sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 16px;
+  font-size: 16px;
```
Comment (samquayle) [inline src/main/resources/static/css/main-ie8.css]: Lots of different font size declarations going on here. Also, why is an IE stylesheet getting -webkit and -moz properties?

**Pull request merged**